File: baekjoon/05-BruteForce/js/2309.js
// 백준 제출 시, readFileSync('/dev/stdin') 으로 변경할 것
const fs = require("fs");
const input = fs.readFileSync("input.txt").toString().split("\n");

const people = input.map((person) => Number(person));
const sum = people.reduce((acc, cur) => acc + cur, 0);

outer_loop: for (let i = 0; i < people.length; i++) {
    for (let j = i + 1; j < people.length; j++) {
        if (sum - (people[i] + people[j]) === 100) {
            ans = people.filter((person) => person != people[i] && person != people[j]);
            ans.sort((a, b) => a - b); // 숫자 오름차순 정렬
            ans.forEach((person) => console.log(person));
            break outer_loop;
        }
    }
}

File: baekjoon/05-BruteForce/python/2309.py
# https://www.acmicpc.net/problem/1463 일곱 난쟁이 (68ms)
import sys
sys.stdin = open('input.txt', 'r')
input = sys.stdin.readline
'''
sum(아홉 난쟁이) - (가짜 난쟁이1 + 가짜 난쟁이2) = 100
'''
people = [int(input().rstrip()) for _ in range(9)]
total = sum(people)

for i in range(len(people)):
    for j in range(i+1, len(people)):
        expt1, expt2 = people[i], people[j]
        if total - (expt1 + expt2) == 100:
            ans = [people[k] for k in range(9) if people[k] != expt1 and people[k] != expt2]
            ans.sort()
            print(*ans, sep='\n')
            exit()